webui/components/ui/icons.tsx

```diff
@@ -4,6 +4,23 @@ import * as React from 'react'
 
 import {cn} from '@/lib/utils'
 
+export function IconEntra({className, ...props}: React.ComponentProps<'svg'>) {
+  return (
+    <svg id="uuid-f8d4d392-7c12-4bd9-baff-66fbf7814b91" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" width="18" height="18" viewBox="0 0 18 18">
```

Comment:
Is it normal to check in the svg's within a typescript file rather than as their own svg referenced here? I see "webui/assets/Microsoft Entra ID color icon.svg" which I assume is the same thing.